src/Compiler/Compiler.cpp

```diff
@@ -1304,6 +1353,14 @@ void Compiler::setTaskManager(TaskManager* newTaskManager) {
     m_taskManager->bindTaskCommand(PLACEMENT_CLEAN, []() {
       GlobalSession->CmdStack()->push_and_exec(new Command("place clean"));
     });
+    m_taskManager->bindTaskCommand(PLACEMENT_SETTINGS, []() {
```

Comment:
why do you need this?

Reply:
yeah I thought that step is necessary for tcl interface but yeah I should be removed. I'll do it.

Reply:
done!